eth/v1alpha1/beacon_chain.proto

```diff
@@ -346,17 +346,20 @@ message ListCommitteesRequest {
 
 message BeaconCommittees {
     message CommitteeItem {
-        // A committee of validator indices that need to attest to beacon blocks.
+        // A committee is a list of validator indices participating in consensus at a slot.
         repeated uint64 committee = 1;
```

Comment:
Should this be renamed? This is a list of validator indices, so why not call it that?

Reply:
Suggestion: `validator_indices`